### ISSUE TITLE
Error: unknown command "generate" for "gowitness"

### ISSUE THREAD

**Issue by ATO7** [j3ssie/Osmedeus#90]
![Screenshot from 2019-09-23 16-49-35](https://user-images.githubusercontent.com/33043331/65436265-328f0980-de22-11e9-961c-5fa884fe8d12.png)

**Comment by j3ssie** [comment]
Yeah **gowitness** update their command. Will fix it soon :D

**Comment by ATO7** [comment]
> Yeah **gowitness** update their command. Will fix it soon :D

Is it affect much on the result ? and also I don't know why when Dirbscan start it takes much time.
by the way it's a great tool. good job bro :+1:

**Comment by j3ssie** [comment]
Not much, I still have aquatone as an alternative screenshot.
For Dirbscan I think I should use `ffuf` with smaller wordlist instead and move bigger wordlist to another routine.

Will update when I free maybe weekend :).

**Comment by j3ssie** [comment]
This should be fixed now

**Comment by 779789571** [comment]
hi,I encountered a problem, and an error occurred during Vulnerability Scanning.（I ran the command "sudo ./osmedeus.py -t xxx.com" on my Mac.）
![image](https://user-images.githubusercontent.com/28771714/77810946-2c7c2d00-70d2-11ea-9d1f-49c7f10ba365.png)
I read this issue and thought that I should change "gowitness generate" to "gowitness report generate". But after the change, I found a new problem. I don't know how to solve it.
![image](https://user-images.githubusercontent.com/28771714/77811025-a8767500-70d2-11ea-9386-9451e5547d2c.png)
